pkg/limayaml/limayaml.go

```diff
@@ -206,6 +207,7 @@ type Probe struct {
 	Mode        ProbeMode // default: "readiness"
 	Description string
 	Script      string
+	Path        string `yaml:",omitempty" json:",omitempty"`
```

Comment:
Shouldn't we consistently use the `File` object?

Reply:
The file objects also adds arch and digest, neither of which is applicable here...

We could rename `Path` to `Location`, to allow for running scripts from URLs?

Reply:
at least digest seems to be applicable here

Reply:
Hmm, probably needs more design then

Reply:
i.e. the wish was to "include" common scripts

If the digest changes every time you edit the script, you still need to copy/paste the sha256?